app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -20,4 +21,20 @@ public function index()
                                 ->paginate(config('define.page_length'));
         return view('backend.categories.index', compact('categories'));
     }
+
+    /**
+     * Update the name corresponding to the category ID in the database.
+     *
+     * @param Illuminate\Http\Request $data data
```

Comment:
please change this comment for params: 
Illuminate\Http\Request $request request